iac/terraform/aws/auth_cognito/lambdas.tf

```diff
@@ -0,0 +1,38 @@
+locals {
+  lambda_archive_path = "${path.module}/../../../../core/services/aws/.build/zips"
+  lambda_options = {
+    custom_email_template = {
+      name          = "custom-email-template"
+      handler       = "customEmailTemplate.default"
+      zip_path      = "${local.lambda_archive_path}/customEmailTemplate.zip"
+      statement     = local.policies.common_policies
+      env_variables = {}
+    },
+    post_confirmation = {
```

Comment:
This lambda name is not very clear

Reply:
Update name now it's cognito_post_confirmation_trigger [Commit](https://github.com/CraftsmenLtd/BloodConnect/commit/28f045306270fdc86a03546bd3b3e08356c4dc74#diff-9fcfda62a75664be8204a906a8560f94308c1ba2b6628a3ee928aacda1a7d7f1)